CMakeLists.txt

```diff
@@ -10,6 +10,8 @@ if (NOT MSVC)
   set(CMAKE_C_STANDARD 11)
 endif()
 
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
```

Comment:
If we're going to set this by default, axe all the places we pass this in explicitly?

```
tools/build_pytorch_libs.bat
204:                  -DCMAKE_EXPORT_COMPILE_COMMANDS=1 ^

tools/build_pytorch_libs.sh
200:                   -DCMAKE_EXPORT_COMPILE_COMMANDS=1 ${CMAKE_ARGS[@]}
304:                   -DCMAKE_EXPORT_COMPILE_COMMANDS=1 \

third_party/onnx/setup.py
157:                '-DCMAKE_EXPORT_COMPILE_COMMANDS=ON',

third_party/onnx-tensorrt/third_party/onnx/setup.py
156:                '-DCMAKE_EXPORT_COMPILE_COMMANDS=ON',
```

Reply:
Oh, I see you did it already.

Reply:
It doesn't seem like you got all the sites.

Reply:
I don't see .bat as a file modified in this diff.